packages/pigment-css-react/README.md

```diff
@@ -347,12 +363,44 @@ This enables you to override the default `color` of the Heading component render
 
 You can also export any styled component you create and use it as the base for additional components:
 
-```tsx
+```jsx
 const ExtraHeading = styled(Heading)({
   // ... overridden styled
 });
 ```
 
+#### Media and Container queries
+
+All Pigment CSS APIs has built-in support for writing media queries and container queries. You can use the `@media` and `@container` keys to define the styles for different screen sizes and container sizes.
+
+```jsx
+import { css, styled } from '@pigment-css/react';
+
+const styles = css({
+  fontSize: '2rem',
+  '@media (min-width: 768px)': {
+    fontSize: '3rem',
+  },
+  '@container (max-width: 768px)': {
+    fontSize: '1.5rem',
+  },
+});
+
+const Heading = styled('h1')({
+  fontSize: '2rem',
+  '@media (min-width: 768px)': {
+    fontSize: '3rem',
+  },
+  '@container (max-width: 768px)': {
+    fontSize: '1.5rem',
+  },
+});
+```
+
+> 💡 **Good to know**:
+>
+> Pigment CSS uses emotion's serialize package behind the scene for turning tagged template and object to CSS string.
```

Comment:
```suggestion
> Pigment CSS uses Emotion's serialize package behind the scenes for turning tagged template and object into CSS strings.
```

What "serialize package" is this? Should we be more specific/link to something here?

Reply:
Or we can just say that we use emotion behind the scene

Reply:
My intention for this is to explicitly tell developers that the styles you write in emotion will work the same as in Pigment. @danilo-leal @samuelsycamore do you have any suggestion?

Reply:
> do we mean to say it works the same as Emotion or that it uses Emotion?

I think the problem is my english 🤣. It uses Emotion that's why the styles also work in Pigment.

Reply:
No worries at all! If we want to cut it simpler, I'd probably suggest something like: 

```suggestion
> Pigment CSS uses Emotion behind the scenes for turning tagged templates and objects into CSS strings.
```

Curious what Sam thinks about it, too!